js_modules/dagit/packages/ui/src/components/Icon.tsx

```diff
@@ -264,19 +272,31 @@ export const Icon = React.memo((props: Props) => {
 });
 
 interface WrapperProps {
-  $color: string;
+  $color?: string;
   $size: number;
   $img: string;
   $rotation: string | null;
 }
 
 export const IconWrapper = styled.div<WrapperProps>`
-  background: ${(p) => p.$color};
   width: ${(p) => p.$size}px;
   height: ${(p) => p.$size}px;
   flex-shrink: 0;
   flex-grow: 0;
-  mask-image: url(${(p) => p.$img});
+  ${(p) =>
+    p.$color == null
```

Comment:
This looks good to me! One thing that might be worth considering is changing `$color?: string` to `$color: string | null` and making this an `===`. It seems like forgetting to add a color with one of the icons that expects to be a mask-image could lead to unintended / somewhat random colors (i'm not 100% sure all the existing svgs use the same color, it's just that they're opaque). Using `color={null}` might be a nice way of saying "yes i explicitly want this to be the native svg color".

You've thought about this more than I have though so I'd defer to what you prefer here!

Reply:
If `color={null}` is passed in we still default to `Colors.Dark`. 

I'll update the type though to use `null` and triple equal it.